zircon.core/src/main/kotlin/org/hexworks/zircon/api/builder/component/TextBoxBuilder.kt

```diff
@@ -66,6 +67,16 @@ data class TextBoxBuilder(
         }
     }
 
+    @JvmOverloads
+    fun addParagraph(paragraph: Paragraph, withNewLine: Boolean = true) = also {
```

Comment:
So I was thinking whether passing a `Builder<Paragraph>` would be better than passing an actual `Paragraph` instance. The reason for this is if the user passes a builder, we have a chance to fix its dimensions, but with a concrete `Paragraph` we do not.